### ISSUE TITLE
Case issue already summit qsos

### ISSUE THREAD

**Issue by Nutster2454** [smurphboy/OARCLogger#8]
'fix' QSOs in wrong case... Add a button?

**Comment by smurphboy** [comment]
Good idea. Not a priority but I'll do it when I look at metrics for QSOs. i.e. show how many are 'bad' and have a feature to fix issues.
